src/api/yorkie.proto

```diff
@@ -293,7 +293,7 @@ message TextNode {
 message RichTextNodeAttr {
     string key = 1;
     string value = 2;
-    TimeTicket updated_at = 3;
+    TimeTicket moved_at = 3;
```

Comment:
This property seems to have been changed incorrectly in Yorkie. Can we change it back to updated_at?
I think we should fix it again in Yorkie.

Reply:
oh sure, I will try to change after Yorkie changed